games/2048.c/Portfile

```diff
@@ -22,3 +22,8 @@ compiler.c_standard     1999
 
 test.run                yes
 test.target             test
+
+# Prevent conflicts with other ports providing 2048 binary
```

Comment:
In other cases where we've done this, we added a note to the Portfile to let the user know that the binary has been installed under a different name. See for example https://github.com/macports/macports-ports/blame/master/sysutils/gdu/Portfile#L40-L42.

I'd suggest we do the same thing here.